Dockerfile

```diff
@@ -2,20 +2,28 @@ FROM kbase/kb_jre:latest as build
 RUN apt-get -y update && apt-get -y install ant git openjdk-8-jdk make
 RUN cd / && git clone https://github.com/kbase/jars
 
-ADD . /src
-RUN cd /src && ant build
-RUN find /src
+COPY . /tmp/data_import_export
+
+ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/
+
+RUN cd /tmp && ln -s /jars && \
+  cd /tmp/data_import_export && \
+  ant war
+
+#ADD . /src
+#RUN cd /src && ant build
+#RUN find /src
+
 FROM kbase/kb_jre:latest
 
 # These ARGs values are passed in via the docker build command
 ARG BUILD_DATE
 ARG VCS_REF
 ARG BRANCH=develop
 
-COPY --from=build /src/deployment/ /kb/deployment/
-COPY --from=build /src/jettybase/ /kb/deployment/jettybase/
-COPY --from=build /src/dist/ /src/dist/
-COPY --from=build /jars /jars
+COPY deployment/ /kb/deployment/
+COPY jettybase/ /kb/deployment/jettybase/
```

Comment:
I see that you're using the templated stuff for /kb/deployment so getting rid of --from=build in the line above is covered. Are you sure you don't need stuff from the jettybase directory as well?

Reply:
I am not sure.  :)  I mostly stole from user profile, which has the same style:

https://github.com/kbase/user_profile/blob/master/Dockerfile#L21-L23

The :dockerize image seems to run fine on CI and next.  I checked in that container, and there isn't much in jettybase/ .  I can try to look in the build image to see if there may be something important there.

Reply:
The java service Dockerfiles are not all consistent.  I think I figured that since user_profile was probably one of the last you converted, it was probably also the cleanest.

Reply:
I don't recall all the dinky files that the build may have dropped in there. I'd leave the --from in there just to be safe.

Reply:
I did a clean build, and it doesn't drop anything extra into jettybase/ .  The only new files are in dist/ (a jar and a war).  I have no objection adding --from=build anyway if you think it's best, since the contents should be the same either way.